docs/3.api/2.components/4.nuxt-link.md

```diff
@@ -123,6 +124,7 @@ defineNuxtLink({
 - **activeClass**: A default class to apply on active links. Works the same as [Vue Router's `linkActiveClass` option](https://router.vuejs.org/api/#linkactiveclass). Defaults to Vue Router's default (`"router-link-active"`)
 - **exactActiveClass**: A default class to apply on exact active links. Works the same as [Vue Router's `linkExactActiveClass` option](https://router.vuejs.org/api/#linkexactactiveclass). Defaults to Vue Router's default (`"router-link-exact-active"`)
 - **prefetchedClass**: A default class to apply to links that have been prefetched.
+- **trailingSlash**: An option to force trailing slash in all urls or remove by default. If option is not defined or different from `"append"`/`"remove"` it will be ignored.
```

Comment:
```suggestion
- **trailingSlash**: An option to either enforce or remove trailing slashes URLs . If  unset or not matching the valid values `"append"` or `"remove"`, it will be ignored.
```